src/features/search/components/CalendarPicker.web.tsx

```diff
@@ -1,48 +1,79 @@
 import { t } from '@lingui/macro'
 import React, { useState, useMemo, useRef, useEffect } from 'react'
+import { isDesktop } from 'react-device-detect'
 import Picker from 'react-mobile-picker'
-import { Animated } from 'react-native'
+import {
+  Calendar as RNCalendar,
+  CalendarTheme,
+  DateObject,
+  LocaleConfig,
+} from 'react-native-calendars'
 import styled from 'styled-components/native'
 
 import {
   monthNamesShort,
   generateDays,
   YEAR_LIST,
+  monthNames,
+  dayNames,
+  dayNamesShort,
+  today,
 } from 'features/bookOffer/components/Calendar/Calendar.utils'
+import { MonthHeader } from 'features/bookOffer/components/Calendar/MonthHeader'
 import { ButtonPrimary } from 'ui/components/buttons/ButtonPrimary'
+import { AppModal } from 'ui/components/modals/AppModal'
 import { Spacer } from 'ui/components/spacer/Spacer'
+import { ArrowNext } from 'ui/svg/icons/ArrowNext'
+import { ArrowPrevious } from 'ui/svg/icons/ArrowPrevious'
+import { Close } from 'ui/svg/icons/Close'
+import { getSpacing } from 'ui/theme'
 
 import { Props } from './CalendarPicker.d'
 
-const Container = styled(Animated.View)({
-  backgroundColor: 'rgba(0,0,0,0.2)',
-  flexDirection: 'column',
-  alignItems: 'stretch',
-  justifyContent: 'flex-end',
-  width: '100vw',
-  height: '100vh',
-  zIndex: 100000,
-  position: 'fixed',
-  top: 0,
-  left: 0,
-  right: 0,
-  bottom: 0,
-})
+LocaleConfig.locales['fr'] = {
+  monthNames,
+  monthNamesShort,
+  dayNames,
+  dayNamesShort,
+  today,
+}
+LocaleConfig.defaultLocale = 'fr'
 
-const CalendarWrapper = styled(Animated.View)({
-  backgroundColor: '#FFF',
-  position: 'relative',
-})
+const renderArrow = (direction: string) => {
```

Comment:
C'est ok avec le typage ca ?
```suggestion
const renderArrow = (direction:'left' | 'right') => {
```

Reply:
Oui normalement tu peux choisir directement des valeurs pour être plus précis que "string". TS sait que c'est du type string du coup et que c'est entre deux valeur.

Reply:
J'ai repris du code existant dans un autre composant de calendrier dédié au offres. Je modifie